c++/src/kj/async-xthread-test.c++

```diff
@@ -49,6 +49,30 @@ inline void delay() { usleep(10000); }
 namespace kj {
 namespace {
 
+// Some trampolines to make TSAN happy about having to read/write volatile values.
+#if defined(_MSC_VER) && !__clang__
+template <typename T>
+inline T xthread_read(volatile T& var) {
```

Comment:
Names should use KJ style.

But I'd be totally fine with just using std::atomic in these unit tests, which I think would be a lot clearer. (The only place where std::atomic bothers me is in public headers... but even there I think I'm coming around to it.)